src/LocationPicker.tsx

```diff
@@ -180,6 +181,26 @@ export default class LocationPicker extends Component<
       map.leafletElement.setView(startPort.center, startPort.zoom);
     }
   };
+  private renderTileLayer = () => {
+    const { geoserver, geoURL, geoLayer } = this.props;
+    const tileLayer = {
```

Comment:
At this point I feel like you can just assign the url and attribution properties directly. Im just going to merge this and do that.